pep-0012.rst

```diff
@@ -438,43 +434,92 @@ Hyperlinks
 ----------
 
 When referencing an external web page in the body of a PEP, you should
-include the title of the page in the text, with either an inline
-hyperlink reference to the URL or a footnote reference (see
-`Footnotes`_ below).  Do not include the URL in the body text of the
-PEP.
+include the title of the page or a suitable description in the text, with
+either an inline hyperlink or a separate explicit target with the URL.
+Do not include bare URLs in the body text of the PEP, and use HTTPS
+links wherever available.
 
 Hyperlink references use backquotes and a trailing underscore to mark
 up the reference text; backquotes are optional if the reference text
-is a single word.  For example::
+is a single word.  For example, to reference a hyperlink target named
+``Python web site``, you would write:
+
+.. code-block:: rst
+
+    In this paragraph, we refer to the `Python web site`_.
+
+which renders as:
 
     In this paragraph, we refer to the `Python web site`_.
 
-An explicit target provides the URL.  Put targets in a References
-section at the end of the PEP, or immediately after the reference.
+.. _Python web site: https://www.python.org/
```

Comment:
Should this line also be part of the RST code block so it's visible as an example?

Reply:
I include it as an example in the paragraph below, where I actually explain how link targets work, so as to not duplicate it as well as potentially confuse users, as I only explain it below.

Reply:
Yup, but actually, I _do_ include it, in the appropriate example where I talk about link targets, which are discussed separately from link sources.